client/blocks/disconnect-jetpack/index.jsx

```diff
@@ -203,7 +203,15 @@ class DisconnectJetpack extends PureComponent {
 		);
 	};
 
-	handleTryRewind = () => this.props.trackTryRewind( this.props.siteSlug );
+	handleTryRewind = () => {
+		const { siteSlug } = this.props;
```

Comment:
You could also destructure `recordTracksEvent` if you want. Or inline `this.props.siteSlug`on L209 since it's only used once.